net/nmap/Portfile

```diff
@@ -31,6 +32,12 @@ checksums      rmd160  8418fcef96848a6ae6bbbeee935b34ce047eab5e \
                sha256  d71be189eec43d7e099bac8571509d316c4577ca79491832ac3e1217bc8f92cc \
                size    11102195
 
+# see https://github.com/nmap/nmap/pull/2732
+patchfiles-append 0001-nmap-7.94-fix-32bit.patch
```

Comment:
Because this patch could be accepted upstream, I have prefixed it with `0001…` so that it is applied before the `0002…` patch.